Changelog: port #14490 entry back to master (#14530)
Automattic/jetpack: readme.txt

* Dashboard: display a notice when a site uses conflicting plans.
* Map Block: improve the calculation and persistence of maps' center points.
* Map Block: improve the generation of an access token on WordPress.com sites.

> * Map Block: update Mapbox GL library to opt into map load based billing.

Reviewer: ```suggestion
* Map Block: support opt-in for usage-based billing.
```

Author: I like the shorter version, but I also like mentioning that this is about the Mapbox library; the billing itself isn't related to Jetpack really.
